whiteboard/whiteboard-ps/whiteboard/Set-WhiteboardSettings.md

```diff
@@ -0,0 +1,78 @@
+---
+external help file: WhiteboardAdmin-help.xml
+Module Name: WhiteboardAdmin
+online version: https://learn.microsoft.com/powershell/module/whiteboard/set-whiteboardsettings
+applicable: Microsoft Whiteboard
+title: Set-WhiteboardSettings
+schema: 2.0.0
+author: shwetawagh
+ms.author: shwetawagh
+ms.reviewer:
+---
+
+# Set-WhiteboardSettings
+
+## SYNOPSIS
+
+Get the users Whiteboard settings.
+
+## SYNTAX
+
+```powershell
+Set-WhiteboardSettings
+ [-ForceAuthPrompt][-Settings]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Sets the tenant settings for the Microsoft Whiteboard services.
+
+## EXAMPLES
+
+### EXAMPLE 1
+
+This command sets the tenant settings for the Microsoft Whiteboard services.
+
+```powershell
+PS C:\> $settings = Get-WhiteboardSettings
+$settings.isEnabledGa = $true
+Set-WhiteboardSettings -Settings $settings
+```
+
+## PARAMETERS
+
+### -ForceAuthPrompt
+
+Always prompt for authentication. Use to ignore cached credentials.
+
+```yaml
+Type: SwitchParameter
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: False
+Accept pipeline input: False
+Accept wildcard characters: False
+Applicable: Microsoft Whiteboard
+```
+
+### -Settings
+
+The object to use as Whiteboard Settings. Should be retrieved via Get-WhiteboardSettings
+
```

Comment:
`-Settings` must have a YAML code block to appear in the Learn article.

Reply:
After you've added an appropriate code block, please remove _Back to submitter_ and reapply _Sign off_.